test_unstructured/chunking/test_title.py

```diff
@@ -642,19 +642,15 @@ def it_rejects_combine_text_under_n_chars_greater_than_maxchars(
             )
 
     def it_does_not_complain_when_specifying_new_after_n_chars_by_itself(self):
-        """Caller can specify `new_after_n_chars` arg without specifying any other options.
-
-        In particular, `combine_text_under_n_chars` value is adjusted down to the
-        `new_after_n_chars` value when the default for `combine_text_under_n_chars` exceeds the
-        value of `new_after_n_chars`.
-        """
+        """Caller can specify `new_after_n_chars` arg without specifying any other options."""
         try:
             opts = _ByTitleChunkingOptions(new_after_n_chars=200)
+            opts._validate()
```

Comment:
better to use `_ByTitleChunkingOptions.new(...)` here instead of explicitly calling `opts._validate()` I think. A user should never have occasion to call the constructor directly.